src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -209,3 +209,55 @@ export const additionalDocumentDescription = () => {
   );
 };
 
+const documentLabels = {
+  1: 'Discharge',
+  2: 'Marriage related',
+  3: 'Dependent related',
+  // 4: 'VA preneed form',
+  5: 'Letter',
+  6: 'Other'
+};
+
+const getVACenterName = (center) => center.treatment.treatmentCenterName;
+const getPrivateCenterName = (release) => release.privateRecordRelease.treatmentCenterName;
+
+const listifyCenters = (center, idx, list) => {
```

Comment:
This is one of those instances of accidental complexity we can't really get around, isn't it? Oh English... :man_shrugging: